src/features/Calendar/Calendar.jsx

```diff
@@ -0,0 +1,24 @@
+import React from "react";
+import PropTypes from "prop-types";
+import CalendarRow from "../CalendarRow/CalendarRow.jsx";
+import "./Calendar.scss";
+export default function Calendar(props) {
+  const { calendarData } = props;
+  return (
+    <table className={"drug-chart-calendar"}>
```

Comment:
If it is a string, you don't need this unnecessary variable wrapping. you can simply do className="string"